submissions/andrewklmn/html-css-popup/app.css

```diff
@@ -0,0 +1,253 @@
+* {
+  padding: 0px;
+  margin: 0px;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  text-align: center;
+  /*
+  background-image: url(img/bg_image.jpg);
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center; 
+  */
+  width: 100vw;  
+  height: 100vh;
+}
+
+label.button {
+  color:black;
+  cursor: pointer;
+  border: white solid thin;
+  box-sizing: border-box;
+}
+
+label.button:hover {
+  color: darkblue;
+  border: gray solid thin;
+}
+
+label#show-items {
+  display: inline-block;
+  height: 36px;
+}
+
+div#main-popup {
+  width:  320px;
+  height: 500px;
+  margin-right: 30px;
+  background-color: white;
+  
+}
+
+nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  z-index: 6;
+}
+
+nav a {
+  color:black;
+  text-align: center;
+  text-decoration: none;
+  font-size: 12px;
+  border-bottom: solid white  1px;
+}
+
+nav a:hover {
+  color: black;
+  text-align: center;
+  border-bottom: solid black 1px;
+}
+
+
+
+
+.topnav-right {
+  width: 250px;
+  height: 48px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-right: 50px;
+}
+
+.icon {
+  width:36px;
+  height:36px;
+}
+
+
+div#container {
+  position: relative;
+  z-index: 5;
+}
+
+div#scroll-area {
+  height:100%;
+  overflow-x: hidden;
+  overflow-y: auto;
+}
+
+input.popup, div.content {
+  display: none;
+}
+input.popup:checked + div.content {
+  display: block;
+}
+
+
+input#more-control:checked + div.content {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+label#button-close {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  z-index: -1;
+  opacity: 0.3;
+  background-color: lightgray;
+  cursor: pointer;
+}
+
+label#more-button {
+  background-color: lightgray;
+  display: inline-block;
+  height: 5%;
```

Comment:
In elements with text content bad idea to use height, because content can be change, but u height is static https://tppr.me/I4ZE2